cmd/flags/auth.go

```diff
@@ -125,19 +115,6 @@ func AuthFlags(f *pflag.FlagSet) {
 	authAPIKeyFile(f)
 }
 
-// AuthOIDCCertKeyFile Use only on generate-token utils
-func AuthOIDCCertKeyFile(f *pflag.FlagSet) {
-	desc := fmt.Sprintf(`OpenID Connect CA key filepath.
```

Comment:
why are you removing this? How can your e2e work? You should only remove the password